salesforce_api/gsheet-new-row-to-sfdc-new-contact/sample.bal

```diff
@@ -0,0 +1,78 @@
+import ballerina/log;
+import ballerinax/googleapis.sheets;
+import ballerinax/salesforce as sfdc;
+
+// Types
+type GSheetOAuth2Config record {
+    string clientId;
+    string clientSecret;
+    string refreshToken;
+    string refreshUrl = "https://www.googleapis.com/oauth2/v3/token";
```

Comment:
Can we use access token approach? Refer to https://github.com/ballerina-guides/integration-samples/blob/main/github-pull-requests-to-gsheets/main.bal#L27

Reply:
We can change the approach to the mentioned one. As the token will expire for google sheets within 1 hour, the existing approach was used.

Reply:
Updated in [900e154](https://github.com/ballerina-guides/integration-samples/pull/31/commits/900e154cf4d53db6e093f4fd6d5b450900bbafa3)

Reply:
@anupama-pathirage is there a specific reason for using the access token directly within the client?